front/src/modules/ui/modal/components/Modal.tsx

```diff
@@ -2,6 +2,18 @@ import React from 'react';
 import styled from '@emotion/styled';
 import { motion } from 'framer-motion';
 
+const StyledContainer = styled.div`
+  align-items: center;
+  color: ${({ theme }) => theme.font.color.primary};
+  display: flex;
+  flex-direction: column;
+  padding: ${({ theme }) => theme.spacing(10)};
+  width: calc(400px - ${({ theme }) => theme.spacing(10 * 2)});
+  > * + * {
+    margin-top: ${({ theme }) => theme.spacing(8)};
```

Comment:
we should avoid that! This will break other components using the modal. The modal should not apply style on its children. We should style this from DeleteWorkspace.tsx.